vmdb/spec/models/ems_openstack_spec.rb

```diff
@@ -44,4 +44,11 @@
     end
   end
 
+  it "event_monitor_options" do
+    MiqEventCatcherOpenstack.stub(:worker_settings => {:amqp_port => 1234})
+    @ems = FactoryGirl.build(:ems_openstack, :hostname => "host", :ipaddress => "::1")
```

Comment:
If the goal is to get away from `ipaddress`, why are we passing it in this spec?
multiple choice:
1) oop
2) t is ok